Nikita_Korzhavin/js9countDown.js

```diff
@@ -0,0 +1,14 @@
+function countDown(num) {
+  for (var i = num; i >= 0; i--) {
+    function time(i) {
```

Comment:
I'd move function declaration outside of for loop

Reply:
fixed